src/app/features/register-student/register-student.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="container">
+    <div class="register-form-students">
+        <form class="form-students" [formGroup]="formRegisterStudent" (ngSubmit)="
+        Send(formRegisterStudent.value)">
+            <div class="example-container">
+                <h1 class="title"> Registro de Estudiantes</h1>
+                <mat-form-field class="seccion-register" appearance="fill">
+                    <mat-label>Identificación Estudiante</mat-label>
+                    <input matInput [name]="formNames.id" [formControlName]="formNames.id" type="number" placeholder="Identificación">
+                    <mat-error *ngIf="formRegisterStudent.get(formNames.id).hasError('required')">Es necesario llenar el campo
+                    </mat-error>
```

Comment:
Use this same suggestion for all the errors
```suggestion
                    <mat-error *ngIf="formRegisterStudent.get(formNames.id).hasError('required')">
                        Es necesario llenar el campo
                    </mat-error>
```